pipeline_dp/private_spark.py

```diff
@@ -46,14 +47,15 @@ def flat_map(self, fn: Callable) -> 'PrivateRDD':
         rdd = self._rdd.flatMapValues(fn)
         return make_private(rdd, self._budget_accountant, None)
 
-    def sum(self, sum_params: aggregate_params.SumParams) -> RDD:
+    def sum(self, sc: SparkContext,
+            sum_params: aggregate_params.SumParams) -> RDD:
         """Computes a DP sum.
 
         Args:
             sum_params: parameters for calculation
         """
 
-        backend = pipeline_dp.SparkRDDBackend()
+        backend = pipeline_dp.SparkRDDBackend(sc)
```

Comment:
I figured out that we don't need to request SparkContext in arguments of `sum()`:

RDD has a field `context`, with corresponding SparkContext.

So we can get context in this line by calling
`self._rdd.context`

Please replace that in this and other functions from PrivateRDD